src/pip/_internal/operations/install/wheel.py

```diff
@@ -63,15 +64,11 @@ def rehash(path, blocksize=1 << 20):
     return (digest, str(length))  # type: ignore
 
 
-def open_for_csv(name, mode):
-    # type: (str, Text) -> IO[Any]
-    if sys.version_info[0] < 3:
-        nl = {}  # type: Dict[str, Any]
-        bin = 'b'
-    else:
-        nl = {'newline': ''}  # type: Dict[str, Any]
-        bin = ''
-    return open(name, mode + bin, **nl)
+def open_for_csv(name, mode, encoding='utf-8'):
```

Comment:
nit: it seems weird to declare `encoding` as a function argument and not use it (as an argument).

Reply:
Yeah I was just inheriting the previous PR for this. But it also seems weird to have a generic name `open_for_csv` but not allowing to change the encoding. Maybe this can be renamed to `open_for_install_record`? (It is the only usage of this function.)

Reply:
> `open_for_install_record`

Sounds reasonable. :)